src/engraving/types/types.h

```diff
@@ -107,6 +107,7 @@ enum class ElementType {
     HOOK,
     LYRICS,
     FIGURED_BASS,
+    FIGURED_BASS_ITEM,
```

Comment:
I believe this line here is causing the unit test failures, all elements later get a higher stacking value, so the failing tests just need to get adjusted